frame/contracts/src/exec.rs

```diff
@@ -1283,14 +1283,8 @@ where
 		}
 		let frame = self.top_frame_mut();
 		let info = frame.terminate();
-		frame.nested_storage.terminate(&info);
-		System::<T>::dec_consumers(&frame.account_id);
```

Comment:
shouldn't this have been removed in the parent PR, as we [moved](https://github.com/paritytech/substrate/pull/14589/files#diff-c1c6c9bef0c70e17c8a7f3124d2e6f87646ae1a1f0ac314c80e9ee3b1d51bf57R594) it to the `charge()` there, and not in this PR?